staging/src/k8s.io/sample-apiserver/hack/update-codegen.sh

```diff
@@ -25,8 +25,8 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-ge
 # --output-base    because this script should also be able to run inside the vendor dir of
 #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
-${CODEGEN_PKG}/generate-internal-groups.sh all \
-  k8s.io/sample-apiserver/pkg/client k8s.io/sample-apiserver/pkg/apis k8s.io/sample-apiserver/pkg/apis \
+${CODEGEN_PKG}/generate-groups.sh all \
+  k8s.io/sample-apiserver/pkg/client k8s.io/sample-apiserver/pkg/apis \
```

Comment:
cc @sttts

Reply:
E.g. we could move internal generation to "{client,informer,lister}-internal" and remove them from the "all" logic. This way there is a way to generate the internal variants for those who need them, but they have to add them manually. Seems to be fair.

Reply:
thanks for review, I added generate-internal-groups back, but only generate defaulter, deep copy and conversion